app/javascript/dashboard/components-next/message/bubbles/Activity.vue

```diff
@@ -14,7 +14,7 @@ const readableTime = computed(() =>
 <template>
   <BaseBubble
     v-tooltip.top="readableTime"
-    class="px-2 py-0.5 !rounded-full flex min-w-0 items-center gap-2"
+    class="px-3 py-1 !rounded-3xl flex min-w-0 items-center gap-2"
```

Comment:
We can still make the border radius smaller. See what's the smallest size the works.

Reply:
@scmmishra Updated to **XL**
<img width="580" alt="image" src="https://github.com/user-attachments/assets/e74c185e-231c-4b9e-a36a-1d096a9c882a" />